hadoop-tools/hadoop-azure/dev-support/testrun-scripts/testsupport.sh

```diff
@@ -63,6 +64,34 @@ fnsBlobConfigFileCheck() {
   fi
 }
 
+if ! command -v az &> /dev/null
+then
+  echo "Azure CLI (az) could not be found. Installing Azure CLI..."
+sudo apt update
+sudo apt install -y azure-cli
+echo "Azure CLI installed successfully."
+fi
+
+uploadToAzure() {
+  azureConfigFilePath="${accountSettingsDir}runresult${accountConfigFileSuffix}"
+  testResultsAccountName=$(xmlstarlet sel -t -v '//property[name = "fs.azure.test.results.account.name"]/value' -n $azureConfigFilePath)
+  accountKey=$(xmlstarlet sel -t -v '//property[name = "fs.azure.account.key"]/value' -n $azureConfigFilePath)
```

Comment:
the account key here is for the test result account right??
And it need not to be same as account for which we are running the CI.
Better rename it similar to `fs.azure.test.results.account.name` to reflect that this is for test result account.

Also, where are these values set??

Reply:
Correct- it's a separate storage account for only saving the test results. Renamed it.
I have created a separate settings file to set these values under our accountSettings directory.